Documentation/markdown/CMakeOptions.md

```diff
@@ -0,0 +1,218 @@
+# CMake Options for the TUM Open Infra Platform.
+This lists all CMake Options that can be set for the TUM Open Infra Platform project.
+
+## Overview of the CMake Options
+
+A CMake Option is visible in the CMake GUI as a box that can be ticked (or not). 
+If you have made changes and want to return to the default values, you will have to delete the CMake cache and configure again. Refer to the CMake "Option" command [here](https://cmake.org/cmake/help/latest/command/option.html).
+
+The options that can be set for the TUM Open Infra Platform are grouped as project-specific options and external options (libraries and thirdparty software). Click on the name below to jump to the respective section. There you will find a description of what each option does, what its default value is and where to find the source code where the option is set if you require more information or wish to make changes there. 
+
+- [Project options](#project-options)
+- [External options](#external-options)
+
+## Project options
+
+The following are options that can be set to specific to the build of the TUM Open Infra Platform project. The options are grouped thematically. Click on the name below to jump to the respective section. 
```

Comment:
Grammar: to specific to the build?

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/111/commits/1fa16dc4d6236c83945a7923a50cf3a42617f5b8